internal/assetloader/loader.go

```diff
@@ -26,18 +26,33 @@ import (
 )
 
 type LoadedAssets struct {
+	ConjureBackcompat            string
```

Comment:
Let's follow model at https://github.com/palantir/godel-conjure-plugin/pull/636/files#diff-34297ac84f12a90cf466c175c10e49a8a0e5836f3a89bccfef5fe3194a6c803cR29 (but single variable instead of slice) -- that makes the asset struct more usable and solves the whole string / string pointer thing we were discussing by bypassing it entirely